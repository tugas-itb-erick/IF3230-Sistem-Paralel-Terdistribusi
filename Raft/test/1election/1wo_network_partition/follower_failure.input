5
set_timeout 2 0
simulate 10
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
crash 1
crash 3
crash 4
crash 5
simulate 100
restart 1
restart 3
restart 4
restart 5
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
simulate 10
get_role 1
get_role 2
get_role 3
get_role 4
get_role 5
end_simulation