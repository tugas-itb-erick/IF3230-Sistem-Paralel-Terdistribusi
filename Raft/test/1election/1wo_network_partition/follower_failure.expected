follower
leader
follower
follower
follower
follower
leader
follower
follower
follower
follower
leader
follower
follower
follower
